src/pkg/packager/deploy.go

```diff
@@ -124,10 +133,12 @@ func (p *Packager) Deploy(ctx context.Context) error {
 	}
 	if len(deployedComponents) == 0 {
 		message.Warn("No components were selected for deployment.  Inspect the package to view the available components and select components interactively or by name with \"--components\"")
+		l.Warn("No components were selected for deployment.  Inspect the package to view the available components and select components interactively or by name with \"--components\"")
```

Comment:
nit: capitalization. admittedly this one is weird cause of the second sentence. I was doing lower first sentence, upper second.